app/controllers/quotes_controller.rb

```diff
@@ -1,14 +1,29 @@
 class QuotesController < ApplicationController
+  before_action :authenticate_user!
   def show
     @quote = Quote.find params[:id]
   end
 
+  def edit
+    @quote = current_user.quotes.find params[:id]
+  end
+
+  def update
+    @quote = current_user.quotes.find params[:id]
+    if @quote.update quote_params
+      redirect_to @quote
+    else
+      render :edit
+    end
+  end
+
   def new
     @quote = Quote.new
   end
 
   def create
     @quote = Quote.new quote_params
+    @quote.user = current_user
```

Comment:
The quote model also has an author on it which is just a string who the quote is attributed to.  Could look into setting something up where the author needs an account or it could be a user that has an account and then they could also have edit/delete control of the quote as well?